sources/zend/Kaltura/Client/ClientBase.php

```diff
@@ -108,6 +108,14 @@ class Kaltura_Client_ClientBase
 	 */
 	private $responseHeaders = array();
 
+	/**
+	 * @var array
+	 */
+	private $supportedFormats = array(
+		self::KALTURA_SERVICE_FORMAT_JSON,
```

Comment:
you probably meant for KALTURA_SERVICE_FORMAT_XML as well

Reply:
fixed